### PR TITLE
curvefs/client: fix lru bug

### LINKED ISSUE #1175: maybe a bug in BucketInitial
```c++
void LeakyBucket::Bucket::BucketInitial(bool burst) {
    if (burst) {
        if (std::fabs(burstLevel) < 0.1) {
            burstBucketEmptyTimes++;
            if (burstBucketEmptyTimes >= FLAGS_bucketEmptyTimes) {
                burstLevelInitial = true;
                burstLevel = burst - static_cast<double>(burst) *
                  FLAGS_bucketLeakIntervalMs /
                    TimeUtility::MilliSecondsPerSecond;
                burstBucketEmptyTimes = 0;
            }
        }
    } else {
        // ...
    }
}
```
local bool burst used for calculate burstLevel, which should be member variable burst ?

### LINKED ISSUE #1190: The log need improve
**Is your feature request related to a problem? (你需要的功能是否与某个问题有关?)**

WechatIMG507

**Describe the solution you'd like (描述你期望的解决方法)**

**Describe alternatives you've considered (描述你想到的折衷方案)**

**Additional context/screenshots (更多上下文/截图)**

### LINKED ISSUE #1215: The hard disk cache is not trimmed after being trimmed several times
**Describe the bug (描述bug)**

**To Reproduce (复现方法)**
1、run vdbench
2、
image
3、The disk cache capacity is always 90%

nbs@pubbeta2-curve2:/data/logs/curvefs$  curve-fuse.log.INFO.20220328-165108.27

**Expected behavior (期望行为)**

**Versions (各种版本)**
OS:
Compiler:
curve-mds:
curve-chunkserver:
curve-snapshotcloneserver:
curve-sdk:
nebd:
curve-nbd:

**Additional context/screenshots (更多上下文/截图)**

### PR DESCRIPTION
**Pull request by wuhongsong**
### What problem does this PR solve?

Issue Number: close #1215  

Issue Number: close #1175   

Issue Number: close #1190

### CODE REVIEW

File: curvefs/src/client/s3/client_s3_cache_manager.cpp [hunk 1]
```diff
@@ -580,7 +580,7 @@ class AsyncPrefetchCallback {
 
         int ret = s3Client_->GetDiskCacheManager()->WriteReadDirect(
             context->key, context->buf, context->len);
-        LOG_IF(ERROR, ret < 0)
+        LOG_IF(INFO, ret < 0)
```
Comment (xu-chaojie) [inline curvefs/src/client/s3/client_s3_cache_manager.cpp:583]: too many log also use LOG(INFO), maybe LOG_EVERY_SECOND will be better.

Comment (baijiaruo) [inline curvefs/src/client/s3/client_s3_cache_manager.cpp:583]: This log is too much, it is meaningless to change to info level, it is recommended to change to vlog

File: curvefs/src/client/s3/disk_cache_manager.cpp [hunk 2]
```diff
@@ -337,27 +337,27 @@ void DiskCacheManager::TrimCache() {
...
             cacheReadFullDir = GetCacheReadFullDir();
             cacheWriteFullDir = GetCacheWriteFullDir();
-
+            std::string cacheKey, cacheKeyBfo, cacheKeyDel;
+            if (!cachedObjName_->GetBack(&cacheKey)) {
+                VLOG(3) << "remove disk file error"
+                        << ", cachedObjName is empty.";
+                continue;
+            }
             while (!IsDiskCacheSafe()) {
```
Comment (xu-chaojie) [inline curvefs/src/client/s3/disk_cache_manager.cpp:350]: use 'do while' will be better, just remove the object from  GetBack before judge IsDiskCacheSafe and GetBefore.

File: curvefs/src/client/s3/client_s3_cache_manager.cpp (hunk 1 above)
Comment (wuhongsong) [reply curvefs/src/client/s3/client_s3_cache_manager.cpp]: fix

Comment (wuhongsong) [reply curvefs/src/client/s3/client_s3_cache_manager.cpp]: fix

File: curvefs/src/client/s3/disk_cache_manager.cpp (hunk 2 above)
Comment (wuhongsong) [reply curvefs/src/client/s3/disk_cache_manager.cpp:350]: GetBefore should before remove object

**Review by baijiaruo — APPROVED** [review]

**Review by xu-chaojie — APPROVED** [review]

**Pull request merged**